Assets/MRTK/Tests/TestUtilities/TestUtilities.cs

```diff
@@ -228,6 +228,53 @@ public static void AssertNotAboutEqual(Quaternion val1, Quaternion val2, string
             Debug.Assert(angle >= tolerance, $"{message}, val1 {val1.ToString("0.000")} almost equals val2 {val2.ToString("0.000")}");
         }
 
+        /// <summary>
+        /// Equvalent to NUnit.Framework.Assert.LessOrEqual, except this also
+        /// applies a slight tolerance on the equality check.
+        /// </summary>
+        /// <remarks>
+        /// This allows for things like LessThanOrEqual(2.00000024, 2.0) to still pass.
+        /// </remarks>
+        public static void AssertLessOrEqual(float observed, float expected, float tolerance = 0.01f)
```

Comment:
Commenting in case others have the same idea I did (using `Mathf.Epsilon` as the default tolerance):

![image](https://user-images.githubusercontent.com/3580640/88326993-7d480f00-ccdb-11ea-9b76-6d9250e54fe6.png) 
![image](https://user-images.githubusercontent.com/3580640/88327133-9cdf3780-ccdb-11ea-9e8b-d51a15a4a580.png)

Bummer